### ISSUE TITLE
Host header

### ISSUE THREAD

**Issue by tarmo888** [Huachao/vscode-restclient#10]
- VSCode Version: 1.3.0
- OS Version: Windows 10.0.10586

```
GET /api/v1/chat HTTP/1.1
Host: localhost:8084
Content-Type: application/x-www-form-urlencoded
```

Steps to Reproduce:
1. Do "Copy request headers" on some call in Chrome Network tab
2. Paste it to VSCode
3. Run REST Client
Result: Invalid URI "/api/v1/chat

Alternative Steps:
1. Copy Generated HTTP code from Chrome Postman extension
2. Paste it to VSCode
3. Run REST Client
Result: Invalid URI "/api/v1/chat

Expected result: Expected it to take the Host from headers if the path is absolute.

**Comment by Huachao** [comment]
@tarmo888 nice catching, I will fix it ASAP. Or if you have interest, you could contribute to this extension , and that's warm welcomed :smiley:

**Comment by Huachao** [comment]
@tarmo888 @vitaliy-komenda the bug as been fixed, and you can try latest version 0.5.5 :sweat_smile:
